app/styles/index.css

```diff
@@ -80,3 +84,9 @@ svg circle {
 #progress-bars {
     margin-top: 15px;
 }
+
+@media screen and (min-width: 767px) {
+    .progress-container {
+        margin: 20px;
+    }
+}
```

Comment:
progress container might be used elsewhere, I don't think changes to the index are necessary.